Gemfile

```diff
@@ -53,22 +56,23 @@ gem 'omniauth-twitter'
 gem 'php-serialization'
 # Manage Tags
 # https://github.com/mbleigh/acts-as-taggable-on/issues/808
-gem 'acts-as-taggable-on', git: 'https://github.com/mbleigh/acts-as-taggable-on.git'
+# gem 'acts-as-taggable-on', git: 'https://github.com/mbleigh/acts-as-taggable-on.git'
+gem 'acts-as-taggable-on', git: 'https://github.com/Fodoj/acts-as-taggable-on', branch: 'rails-5.2'
 # Complicated Search
 gem 'ransack'
 # Manage Title and meta tags
 gem 'meta-tags'
 gem 'sitemap_generator'
 # AWS!!
-gem 'aws-sdk', '~> 2.6'
-gem 'aws-sdk-core'
-gem 'aws-sdk-resources'
+# gem 'aws-sdk', '~> 2.6'
+# gem 'aws-sdk-core'
+# gem 'aws-sdk-resources'
+
+gem 'aws-sdk', '~> 3'
+# gem "aws-sdk-s3", require: false
 # Azure
-gem 'azure', '= 0.7.6'
-# gem 'azure-contrib', git: 'https://github.com/dmichael/azure-contrib.git'
-gem 'azure-contrib', git: 'https://github.com/ryuzee/azure-contrib.git', ref: 'b74076c'
-gem 'azure-core', '= 0.1.5'
-# gem 'azure-storage', '~> 0.12.0'
+gem 'azure'
+gem "azure-storage"
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.


<!-- Comment by [SideCI](https://sideci.com) -->